src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -746,7 +746,8 @@ class EltwiseJitExecutor : public Eltwise::IEltwiseExecutor {
                               const float alpha,
                               const float beta,
                               const float gamma,
-                              const std::vector<ov::element::Type>& input_precisions = {}) {
+                              const std::vector<ov::element::Type>& input_precisions = {},
+                              const std::vector<ov::element::Type>& output_precisions = {}) {
```

Comment:
```suggestion
                              const ov::element::TypeVector& input_precisions = {},
                              const ov::element::TypeVector& output_precisions = {}) {
```
There is ov alias for this kind of vector